### DISCUSSION TITLE
Problems with Mapbox and DeckGl

### DISCUSSION THREAD

**Discussion by angelc16** [visgl/deck.gl#8184] [Q&A]
I am using bitmaplayer and fill-extrusion layer to represent buildings with a roof and an image on top, the layers have different heights so they don't overlap.
It looks good but in certain screen proportions or when changing the browser zoom or moving the map, some bitmaplayer layers, the farthest from the screen center begin to cut or disappear, as in the gif I attach.
Is this a bug or should I change some settings? I would appreciate your guidance on this issue.
Mapbox Version: 2.4.0
DeckGl Version: 8.9.30
https://codepen.io/angelc37/pen/JjweqEW
https://imageupload.io/4Gxu2foP8gyt1w4

*Updated Deckgl Version

**Comment by Pessimistress** [comment]
You are using a deck.gl version that is no longer maintained. Multiple bug fixes regarding Mapbox integration has been released since then.

**Reply by angelc16** [reply]
Thanks for the clarification. I have updated the deck.gl version, but unfortunately the problem remains. Do you have any additional suggestions to address this situation?"
